roles/reset/tasks/main.yml

```diff
@@ -264,6 +264,8 @@
   with_items:
     - "{{ kube_config_dir }}"
     - /var/lib/kubelet
+    - /var/lib/containerd
```

Comment:
`"{{ containerd_storage_dir }}"` can be used instead for specifying the other value than the default `/var/lib/containerd`.